pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/source/TopicSchema.java

```diff
@@ -44,6 +45,8 @@
 @Slf4j
 public class TopicSchema {
 
+    public static final String JSR_310_CONVERSION_ENABLED = "jsr310ConversionEnabled";
+    public static final String ALWAYS_ALLOW_NULL = "alwaysAllowNull";
```

Comment:
I don't find any references to these fields on the master branch now. @315157973 do you remember why we add these lines here?